### PR TITLE
Sourav added his name.

### PR DESCRIPTION
**Pull request by snayak4379**

### CODE REVIEW

File: name-list.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
-list = [];
+list = ["Sourav"];
 name = "";
 for(i = 0; i < list.length; i++){
-  if(list[i] == name){
```
Comment (jsquirrelz) [inline name-list.js:4]: Why did you remove this extra space?

**Comment by jsquirrelz** [comment]
👍 :shipit:

File: name-list.js (hunk 1 above)
Comment (katietheiss) [reply name-list.js:4]: Yes, why did you do that?!!
